frontend/__tests__/components/CardCarousel.test.tsx

```diff
@@ -34,21 +34,25 @@ describe('CardCarousel', () => {
     expect(screen.queryAllByRole('img')).toHaveLength(1)
   })
 
-  test('clicking on card buttons sets placeholder status', async () => {
+  test('clicking on decline button sets placeholder status', async () => {
     render(<CardCarousel restaurants={[FAKE_RESTAURANT]} />)
 
     expect(screen.getByText('Button status: none')).toBeVisible()
 
-    await userEvent.click(screen.getByRole('button', { name: 'Accept' }))
-
-    expect(
-      screen.getByText('Button status: accept button clicked'),
-    ).toBeVisible()
-
     await userEvent.click(screen.getByRole('button', { name: 'Decline' }))
 
     expect(
       screen.getByText('Button status: decline button clicked'),
     ).toBeVisible()
   })
+
+  test('clicking on accept button calls the liked function', async () => {
+    const likeRestaurant = jest.fn()
```

Comment:
nitpicking:  I'd name this something to be more consistent with how it's used. e.g. `addLikedRestaurantMock`.